mtags/src/main/scala-3/scala/meta/internal/pc/completions/MatchCaseCompletions.scala

```diff
@@ -209,8 +198,15 @@ object CaseKeywordCompletion:
       completionPos,
       clientSupportsSnippets,
     )
+    val isFromPartial = parent match
+      case EmptyTree => false
+      case _ => true
```

Comment:
`isFromPartial` is used only once as condition in `if`. It might be better to replace that if with pattern match itself.
```scala
    parent match
      // ABC scenario
      case EmptyTree =>
        .. code ...
      // DFG scenario
      case _ =>
        .. code ...
```